### PR TITLE
Moved Fortigate to Package format

### PR DESCRIPTION
**Pull request by guykeller**
## Status 
Ready

## Description
Moved Fortigate to Package format

## Does it break backward compatibility?
   - No

### CODE REVIEW

**Comment by Itay4** [comment]
please rename to `FortiGate`

**Comment by guykeller** [comment]
Why renaming to FortiGate?
It was "Fortigate" way before I added new content.
Moreover - this PR is dedicated strictly to moving the integration to a package, which as far as I know should be named like the integration was before

File: Integrations/FortiGate/FortiGate.py [hunk 1]
```diff
@@ -0,0 +1,1025 @@
+import demistomock as demisto
+from CommonServerPython import *
+''' IMPORTS '''
+
+import json
+import requests
+from typing import *
```
Comment (yaakovi) [inline Integrations/FortiGate/FortiGate.py]: why?

Comment (guykeller) [reply Integrations/FortiGate/FortiGate.py]: removed

**Pull request merged**